### DISCUSSION TITLE
XAML intellisense after removing UWP project

### DISCUSSION THREAD

**Discussion by tsorger1403** [unoplatform/uno#8900] [Q&A]
I have created a project with the [Uno/Prism template](https://platform.uno/docs/articles/get-started-dotnet-new.html?tabs=windows#uno-platform-blank-prism-application). After some modifications I have removed the UWP project and replaced it with a desktop project, but now I don't have XAML intellisense anymore. According to this [Get Started](https://platform.uno/docs/articles/get-started-vs.html?tabs=windows#make-sure-xaml-intellisense-is-enabled), XAML intellisense is supported when the UWP head is active (which is/was correct).
Is there a way to support XAML intellisense even though there is no UWP project anymore?

**Comment by jeromelaban** [comment]
The Uno/Prism template uses an older version of Uno which does not contain some of the improvements for intellisense for Visual Studio for Windows made in Uno 4.3.6. 

There's currently no public version of Prism which supports Uno 4, but you can see with the Prism maintainers on how to get a compatible version.
